JS.md

```diff
@@ -215,11 +214,11 @@ has been modified so that each export simply has a `name`, `type` and `index`:
   * If the `type` is function, then export an 
     [Exported Function Exotic Object](#exported-function-exotic-objects),
     reusing an existing object if one exists for the given function definition,
-    otherwise creating a new object.
+    otherwise creating a new object given `instance` and `index`.
   * If the `type` is global:
-    * If the global is not immutable, then throw a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
+    * Assert: the global is immutable by MVP validation constraint.
```

Comment:
Oh here I just meant it in the "this is a condition that should always be true because it is checked elsewhere".  I think this is the same [meaning as in ecma262](https://tc39.github.io/ecma262/#assert).